internal/pkg/daemon/bpfrecorder/bpf/recorder.bpf.c

```diff
@@ -69,18 +76,38 @@ int sys_enter(struct trace_event_raw_sys_enter * args)
     // Update the command name if required
     char comm[MAX_COMM_LEN];
     bpf_get_current_comm(comm, sizeof(comm));
-    if (bpf_map_lookup_elem(&comms, &comm) == NULL) {
+    if (bpf_map_lookup_elem(&comms, &pid) == NULL) {
         bpf_map_update_elem(&comms, &pid, &comm, BPF_ANY);
     }
 
     // Update the syscalls
-    u8 * const syscall_value = bpf_map_lookup_elem(&syscalls, &pid);
-    if (syscall_value) {
-        syscall_value[syscall_id] = 1;
+    u8 * const mntns_syscall_value = bpf_map_lookup_elem(&mntns_syscalls, &mntns);
+    if (mntns_syscall_value) {
+        mntns_syscall_value[syscall_id] = 1;
+        u8 * isRecord;
+        isRecord = bpf_map_lookup_elem(&mntns_record, &mntns);
+        if (isRecord == NULL) {
+            // New element, throw event
```

Comment:
I think you should update the `mnts_record` map somewhere in here. It doesn't seem to get updated.

Reply:
I want to update mnts_record in user mode.